### DISCUSSION TITLE
Request tutorial on making transverse flutes

### DISCUSSION THREAD

**Discussion by edwardkort** [edwardkort/WWIDesigner#94] [Flute Study] [answered]
From Ryan Fisher:
 I have just download WWID and would also benefit greatly from a tutorial on how to use it to create transverse flutes, there seems to be quite a lack of information on how to make transverse flutes such as the Chinese Dizis like the Qu di, I have scowered the internet, and it seems your WWID is my last attempt before throwing in the towel.

**Comment by bhp1** [comment, marked as answer]
We can't offer much guidance on _making_ flutes, particularly not the more exotic ones like dizis.  There are resources available.  One that I know of is Doug Tipples's https://tippleflutes.com/making-a-simple-irish-flute/.
